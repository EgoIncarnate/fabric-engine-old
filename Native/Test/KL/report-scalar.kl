operator entry()
{
  report 3.141;
}
