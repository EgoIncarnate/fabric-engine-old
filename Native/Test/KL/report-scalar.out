GlobalList
  Function( 'entry' ('entry'), void )
    ParamList
    CompoundStatement
      StatementList
        Report
          ConstScalar( 3.141 )
3.141
